src/main/kotlin/com/terraformation/backend/accelerator/api/ParticipantProjectSpeciesController.kt

```diff
@@ -30,18 +66,34 @@ class ParticipantProjectSpeciesController(
   ): GetParticipantProjectSpeciesResponsePayload {
     val model = participantProjectSpeciesStore.fetchOneById(participantProjectSpeciesId)
 
-    return GetParticipantProjectSpeciesResponsePayload(
-        ParticipantProjectSpeciesPayload(
-            feedback = model.feedback,
-            id = model.id,
-            projectId = model.projectId,
-            rationale = model.rationale,
-            speciesId = model.speciesId,
-            submissionStatus = model.submissionStatus,
-        ))
+    return makeGetResponse(model)
   }
+
+  private fun makeGetResponse(
```

Comment:
This is totally fine. An alternate way to do it would be to add a secondary constructor to `GetParticipantProjectResponsePayload` that takes the model and constructs the inner payload.

Reply:
Thanks, I am leaving this for now because I suspect I will need to include the full `species` model with the response, and I was following the pattern a bit from `ParticipantsController`